### ISSUE TITLE
Feature: Send terminal window size info

### ISSUE THREAD

**Issue by tom95** [hpi-swa-lab/sb-terminal#1]
At the moment, e.g. `ls` puts all output in a single column. I assume this is because the terminal doesn't tell the pty how large it is.

**Comment by s4pu** [comment]
Yeah, I noticed that as well. One way to manipulate the output would be using `stty`, e.g. `stty cols 30`.
